ni/__main__.py

```diff
@@ -24,6 +26,9 @@ def handler(create_client: Callable[[], aiohttp.ClientSession], server: ni_abc.S
                 contribution = await ContribHost.process(server, request, client)
                 usernames = await contribution.usernames()
                 server.log("Usernames: " + str(usernames))
+                usernames = [username
+                                for username in usernames
+                                    if username not in os.environ.get('WHITELISTED_USERS')]
```

Comment:
- ``os.environ.get('WHITELISTED_USERS')`` might be ``None``

- the list of users in the environment variable isn't getting split on commas, so a candidate might be matched against a part of a name in the whitelist

Reply:
Thanks! I updated this with unittests, and also updated the readme with instructions on how to add users to the ignored list.

Reply:
Operationally, LGTM now.

The name ``get_not_ignored_usernames`` feels somewhat tortuous; perhaps something like ``get_checked_usernames`` would be more readable?  Not a necessary change, though.

Reply:
`get_checked_usernames` sounds good.

I renamed it.